roles/openshift_logging_elasticsearch/library/openshift_logging_elasticsearch_facts.py

```diff
@@ -0,0 +1,351 @@
+'''
```

Comment:
The idea is that upgrades are done per elasticsearch cluster, so facts for upgrade should be gathered for individual elasticsearch cluster.